### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by rcronin** [mariusandra/insights#110]
Is this project still in development?

**Comment by mariusandra** [comment]
Not really. All my energy for the last 2-3 years has gone into https://github.com/posthog/posthog - which is a similar yet different open source product analytics tool.
